basis-library/text/char.sig

```diff
@@ -12,36 +12,36 @@ signature CHAR =
 
       eqtype string
 
-      val minChar: char
-      val maxChar: char
-      val maxOrd: int
-      val succ: char -> char
-      val pred: char -> char
-      val < : char * char -> bool
-      val <= : char * char -> bool
-      val > : char * char -> bool
-      val >= : char * char -> bool
-      val compare: char * char -> order
-      val contains: string -> char -> bool
-      val notContains: string -> char -> bool
-      val toLower: char -> char
-      val toUpper: char -> char
-      val isAscii: char -> bool
-      val isAlpha: char -> bool
-      val isAlphaNum: char -> bool
-      val isCntrl: char -> bool
-      val isDigit: char -> bool
-      val isGraph: char -> bool
-      val isHexDigit: char -> bool
-      val isLower: char -> bool
-      val isUpper: char -> bool
-      val isPrint: char -> bool
-      val isPunct: char -> bool
-      val isSpace: char -> bool
-
-      val toString: char -> String.string
+      val minChar: char 
+      val maxChar: char 
+      val maxOrd: int 
+      val succ: char -> char 
+      val pred: char -> char 
+      val < : char * char -> bool 
+      val <= : char * char -> bool 
+      val > : char * char -> bool 
+      val >= : char * char -> bool 
+      val compare: char * char -> order 
+      val contains: string -> char -> bool 
+      val notContains: string -> char -> bool 
+      val toLower: char -> char 
+      val toUpper: char -> char 
+      val isAscii: char -> bool 
+      val isAlpha: char -> bool 
+      val isAlphaNum: char -> bool 
+      val isCntrl: char -> bool 
+      val isDigit: char -> bool 
+      val isGraph: char -> bool 
+      val isHexDigit: char -> bool 
+      val isLower: char -> bool 
+      val isUpper: char -> bool 
+      val isPrint: char -> bool 
+      val isPunct: char -> bool 
+      val isSpace: char -> bool 
+      
+      val toString: char -> String.string 
       val scan: (Char.char, 'a) StringCvt.reader -> (char, 'a) StringCvt.reader
-      val fromString: String.string -> char option
+      val fromString: String.string -> char option 
```

Comment:
These changes to `basis-library/text/char.sig` seem to be purely white space changes and unrelated to the refactoring.  We should avoid unnecessary white space changes.